docs/user-guide/creating-a-driver.md

```diff
@@ -249,3 +280,13 @@ The Helm chart will install:
 - an Ingress rule to expose the APIs to external clients from the Kuberentes using an ingress controller (if your cluster has one installed)
 
 This implementation of a Helm chart should allow you deploy your driver to Kubernetes right away! However, you are free to adjust this directory to produce a chart more suited to your deployment patterns. There is no requirement to keep this directory, your Python application is unaffected by it's presence.
+
+To build the Helm chart consult the `devdocs` included in your driver.
+
+# Next Steps
+
+Ensure you have read through [The Basics of a Driver](#the-basic-of-a-driver) to become familiar with the contents of your driver project. 
```

Comment:
Should this be "the-basics-of-a-driver" i.e. add an "s"?

Reply:
Yep. Will change.